integration_tests/__tests__/__snapshots__/failures-test.js.snap

```diff
@@ -103,3 +103,272 @@ Ran all test suites matching \\"assertion-count-test.js\\".
 ",
 }
 `;
+
+exports[`works with node assert 1`] = `
+Object {
+  "rest": " FAIL  __tests__/node-assertion-error-test.js
+  ● assert
+
+    assert.equal(received, expected) or assert(received) 
+    
+    Expected value to be (operator: ==):
+      true
+    Received:
+      false
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:16:3)
+
+  ● assert with a message
+
+    assert.equal(received, expected) or assert(received) 
+    
+    Expected value to be (operator: ==):
+      true
+    Received:
+      false
+    
+    Message:
+      this is a message
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:20:3)
+
+  ● assert.ok
+
+    assert.equal(received, expected) or assert(received) 
+    
+    Expected value to be (operator: ==):
+      true
+    Received:
+      false
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:24:10)
+
+  ● assert.ok with a message
+
+    assert.equal(received, expected) or assert(received) 
+    
+    Expected value to be (operator: ==):
+      true
+    Received:
+      false
+    
+    Message:
+      this is a message
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:28:10)
+
+  ● assert.equal
+
+    assert.equal(received, expected) or assert(received) 
+    
+    Expected value to be (operator: ==):
+      2
+    Received:
+      1
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:32:10)
+
+  ● assert.notEqual
+
+    assert.notEqual(received, expected)
+    
+    Expected value not to be (operator: !=):
+      1
+    Received:
+      1
+    
+    Difference:
+    
+    Compared values have no visual difference.
+      
+      at Object.<anonymous> (__tests__/node-assertion-error-test.js:36:10)
+
+  ● assert.deepEqual
+
+    assert.deepEqual(received, expected)
+    
+    Expected value to deepEqual to:
```

Comment:
Is this just like ==, but deep? In that case, how about:

"Expected value to deeply equal:"

maybe?